horizon/enforcer/opa/config_maker.py

```diff
@@ -1,28 +1,23 @@
-import os
+from pathlib import Path
 
 import jinja2
 from opal_common.logger import logger
 
 from horizon.config import SidecarConfig
 from horizon.startup.api_keys import get_env_api_key
 
+TEMPLATES_PATH = Path(__file__).parent.parent / "static" / "templates"
```

Comment:
I think you should have another .parent here

Reply:
Few commits after had fixed that, it was a bug back then too 🙂